onboard/catkin_ws/src/motion_planning/scripts/move_tasks.py

```diff
@@ -11,30 +11,34 @@ class MoveToPoseGlobalTask(Task):
     def __init__(self, x, y, z, roll, pitch, yaw, *args, **kwargs):
         super(MoveToPoseGlobalTask, self).__init__(*args, **kwargs)
 
-        self.desired_pose = Pose()
-        self.desired_pose.position.x = x
-        self.desired_pose.position.y = y
-        self.desired_pose.position.z = z
-        self.desired_pose.orientation = Quaternion(*quaternion_from_euler(roll, pitch, yaw))
+        self.desired_pose = Pose(
+                            position=Point(x=x, y=y, z=z),
+                            orientation=Quaternion(
+                                *quaternion_from_euler(roll, pitch, yaw)))
```

Comment:
I think this is clearer to read
```suggestion
        self.desired_pose = Pose()
        self.desired_pose.position = Point(x=x, y=y, z=z)
        self.desired_pose.orientation = Quaternion(*quaternion_from_euler(roll, pitch, yaw))
```

Reply:
okay, fair enough